Resources/public/js/views/services/ez-viewservice.js

```diff
@@ -193,7 +193,18 @@ YUI.add('ez-viewservice', function (Y) {
              */
             app: {
                 writeOnce: "initOnly"
-            }
+            },
+
+            /**
+             * The config is an element configured for the route
```

Comment:
I would put something like `The configuration coming from the matched route in the application.`